content/en/blog/2023/jmx-metric-insight/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Gaining JMX Metric Insights with the OpenTelemetry Java Agent
+linkTitle: JMX Metric Insight
+date: 2023-01-11
```

Comment:
leaving this here: before going live/merging the PR we need to set the correct date.